Legion/src/Assets/animation.cpp

```diff
@@ -90,7 +106,41 @@ void RpakLib::ExportAnimationRig(const RpakLoadAsset& Asset, const string& Path)
 		skelOut.write(skelBuf, SkeletonHeader.DataSize);
 		skelOut.close();
 
-		// todo: rseq
+		const uint64_t ReferenceOffset = this->GetFileOffset(Asset, RigHeader.AnimationReferenceIndex, RigHeader.AnimationReferenceOffset);
+
+		for (uint32_t i = 0; i < RigHeader.AnimationReferenceCount; i++)
+		{
+			RpakStream->SetPosition(ReferenceOffset + ((uint64_t)i * 0x8));
+
+			uint64_t AnimHash = Reader.Read<uint64_t>();
+
```

Comment:
you should check if the animation is loaded before exporting it, as otherwise you will crash when dealing with non-local assets or DFS'd animations (when guids are replaced by `0xDF5` in this array)
```suggestion
if(Assets.ContainsKey(AnimHash))
```